MIP/mip-74/README.md

```diff
@@ -0,0 +1,165 @@
+
+#### Insurance funds
+
+ We assume there is a Insurance Fund on both L1 and L2, with values `insurance_fund_L1` and `insurance_fund_L2`, respectively.
+
+> [!NOTE]
+> These values are considered constant in the sequel. There may be updated if needed or if new funds are added to the pools.
+
+The Insurance Fund rate-limits the transfers i.e., for a given transfer from source chain to target chain the Insurance Fund on the _target chain_ is responsible for the rate limit, and thus we will refer to the `insurance_fund_target`.
+For a transfer from L1 (L2) to L2 (L1) the `insurance_fund_target = insurance_fund_L2` (`insurance_fund_L1`) is responsible for the rate limit.
+
+#### Rate limit on the target chain
+
+The rate limit is dependent on the fund size in the Insurance Fund. In particular the maximum rate limit is defined by
+
+`max_rate_limit_target = insurance_fund_target / reaction_time`,
+
+where the `reaction_time` is the time it takes for the Governance Operator to react to a faulty or compromised component. The `reaction_time` is a parameter that is set by the Governance Operator. The Governance Operator MAY set the actual rate limit lower than the `max_rate_limit_target`. However the Rate Limiter MUST NOT set the rate limit higher than the `max_rate_limit_target`.
+
+The rate limit MAY also be adjusted by the Governance Operator.
+
+`rate_limit_target = rate_reduction_target * max_rate_limit_target`,
+
+where `rate_reduction_target` $\in$ `[0,1]` is a parameter that is set by the Governance Operator. Note the `rate_limit_target` MUST not be larger than `max_rate_limit_target`.
+
+The following are possible ways to adjust the rate limit:
+
+1. The Governance Operator can adjust the rate limit by adding or removing funds from the Insurance Fund.
+1. The Governance Operator may adjust the rate limit by changing the `reaction_time`.
+1. The Governance Operator may adjust the rate limit by changing the `rate_reduction_target`.
+
+#### Rate limit on the source chain
+
+On the source chain the rate limit MAY be lowered by the Relayer. This is to ensure that the rate limit on the target chain is not exceeded. It also permits the Relayer to catch up in case of the Relayer has been down for some time.
+
+`rate_limit_source = min{rate_reduction_source * rate_limit_target, rate_limit_operator_source}`,
+
+where `rate_reduction_source` $\in$ `[0,1]` is a parameter that is set by the Relayer. `rate_limit_operator_source` is a parameter that is set by the Governance Operator. Note the `rate_limit_source` SHOULD not be larger than `rate_limit_operator_source`.
+
+### Rate limitation adjustment algorithm
+
+The rate limitation works as follows:
+
+> [!NOTE]
+> I can convert the following into pseudo code, after we have discussed the algorithm and it makes sense.
+
+**Algorithm for the Native Bridge contract on the source chain**
```

Comment:
This assumes to use MIP-58's [design C](https://github.com/movementlabsxyz/MIP/blob/d27459f7309b1fb196dce6d50d64bda387f9cf6c/MIP/mip-58/README.md#4-rate-limiting-designs), correct?

Reply:
yes, and this this MIP is now referenced in MIP74 at the beginning of `Section #### 4 **Rate limiting designs**`